helm/hpcc/templates/thor.yaml

```diff
@@ -81,6 +88,7 @@ spec:
                 {{ printf "\"--name=%s\"" .name }}
               ]
 {{- include "hpcc.addSecurityContext" (dict "root" $ "me" .) | indent 8 }}
+{{- include "hpcc.addResources" (dict "me" $thorScope.agentResources) | indent 8 }}
```

Comment:
Actually, if useChildProcesses is in this context, it would spawn master and slaves as child processes.
Turning on useChildProcesses here is a developer only option really, there are some known issues with it.

Should generate stub resources when off, and when on... all bets are off! (it would need to be sum total of master + N * workers to make any sense)